src/main/kotlin/com/terraformation/backend/accelerator/db/DeliverableDueDateStore.kt

```diff
@@ -0,0 +1,139 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.accelerator.model.DeliverableDueDateModel
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.CohortId
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.accelerator.ModuleId
+import com.terraformation.backend.db.accelerator.tables.references.COHORT_MODULES
+import com.terraformation.backend.db.accelerator.tables.references.DELIVERABLES
+import com.terraformation.backend.db.accelerator.tables.references.DELIVERABLE_COHORT_DUE_DATES
+import com.terraformation.backend.db.accelerator.tables.references.DELIVERABLE_PROJECT_DUE_DATES
+import com.terraformation.backend.db.default_schema.ProjectId
+import jakarta.inject.Named
+import java.time.LocalDate
+import org.jooq.DSLContext
+import org.jooq.impl.DSL
+
+@Named
+class DeliverableDueDateStore(
+    private val dslContext: DSLContext,
+) {
+  fun fetchDeliverableDueDates(
+      cohortId: CohortId? = null,
+      moduleId: ModuleId? = null,
+      deliverableId: DeliverableId? = null,
+  ): List<DeliverableDueDateModel> {
```

Comment:
Maybe it'll become clearer after I see how this is going to be used, but I was a little surprised there wasn't a `projectId` argument here for cases where you only care about a single project.

Reply:
The only use case for now is really to display the information for editing in the admin UI. The due date for a deliverable for a project is already part of the DeliverableSubmissionStore. 

My thinking is that this is an "internal" store where admin will get to see more details on how the due date comes about. External users will only get to see the final result if that makes sense.